test/functional/apps/discover/_discover.ts

```diff
@@ -122,7 +122,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           return actualRenderingCount === prevRenderingCount + 6;
         });
         const newDurationHours = await PageObjects.timePicker.getTimeDurationInHours();
-        expect(Math.round(newDurationHours)).to.be(26);
+        expect(Math.round(newDurationHours)).to.be(27);
```

Comment:
Changes to `EuiPage` increased padding on some containers, resulting in a slightly smaller histogram container, resulting in a fractionally larger brush area given the same absolute dimensions.